templates/emails/transactional/spoof-reply.html

```diff
@@ -9,6 +9,16 @@ <h1>
   </h1>
 {% endcall %}
 
+{% call text() %}
+- From: <b>{{ alias }}</b>
+<br />
+- To: <b>{{ to_email }}</b>
+<br />
+- Subject: <b>{{ subject }}</b>
```

Comment:
I think we don't have information like subject or to_email, you can see where this template is used on https://github.com/simple-login/app/blob/require-sudo-on-account-change/app/handler/dmarc.py#L173